doc/source/whatsnew/v0.16.0.txt

```diff
@@ -22,6 +22,10 @@ New features
 
 .. _whatsnew_0160.api:
 
+- The ``read_excel()`` function's sheetname argument now accepts a list and None, to get 
+  multiple or all sheets respectively.  If used, a dictionary is returned.
+- A ``verbose`` argument has been added to ``io.read_excel()``, defaults to False.
```

Comment:
[x] explain the purpose of verbose. (Done, need to confirm links with sphinx build)
[ ] add the issue number to this. (there was no issue number, so?)
[x] add a link to the documentation (Done, need to confirm links with sphinx build)
[x] mini-example (done, copied + pasted)
...commit coming.